be/src/olap/lru_cache.cpp

```diff
@@ -622,20 +629,27 @@ uint64_t ShardedLRUCache::new_id() {
     return _last_id.fetch_add(1, std::memory_order_relaxed);
 }
 
-int64_t ShardedLRUCache::prune() {
-    int64_t num_prune = 0;
+std::tuple<PrunedCount, PrunedSize> ShardedLRUCache::prune() {
```

Comment:
use range_view algorithm replace for

Reply:
![Uploading img_v3_0288_e473f88a-b82e-4289-a928-8efae267ffbg.jpg…]()